### ISSUE TITLE
"Install completed successfully." buttons unreadable in Light theme

### ISSUE THREAD

**Issue by illume** [kubernetes-sigs/headlamp#3157]
## Describe the bug

These links are not visible in Light theme.

![Image](https://github.com/user-attachments/assets/7e6f0077-5694-443a-acb4-242062c576fb)

## To Reproduce

Use the app.
Go to plugin catalog.
Install minikube plugin.
See unreadable text.

## Environment (please provide info about your environment):
WSL

**Comment by skoeva** [comment]
Fixed by https://github.com/headlamp-k8s/plugins/pull/228
